tests/console/start_systemd_testkit_offline.pm

```diff
@@ -0,0 +1,109 @@
+# SUSE's openQA tests
+#
+# Copyright 2020 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: install and execute the external systemd and installation must
```

Comment:
I think it should be phrased a bit better and add the link of the progress ticket, which contains more information.

Reply:
Sure Anna, I have added bit of summary and linked ticket. Thank you :)